DiamondLightSource/python-artemis
# 572 deploy dodal as repo in prod fixed

Fixes #572  and #627

### To test:
1. Run "deploy_artemis.py dev" and confirm issue requirements are satifsfied, and that the symlink is correct
2. Confirm beamline deployment location is correct
